pkg/manifests/manifests.go

```diff
@@ -1719,6 +1719,18 @@ func (f *Factory) PrometheusAdapterConfigMapPrometheus() (*v1.ConfigMap, error)
 	return cm, nil
 }
 
+func auditLogsEnabled(level string) bool {
+	switch level {
+	case "None",
+		"Metadata",
+		"Request",
+		"RequestResponse":
+		return level != "None" // no audit logs for None
+	default:
+		return false
+	}
```

Comment:
this function seems a bit confusing to me, since it checks a list of magic strings without any context that are essentially ignored, perhaps it could just be 

```	
if level == "" || level == "None" {
		return false
	}
```

Reply:
`None, Metadata, Request, RequestResponse` are the only valid `Log Levels` for which an audit-policy is defined. 
 

```
if level == "" || level == "None" { return false }
```
the above would return `true` if user sets the loglevel to "foobar", wouldn't it?  or am I missing something?

Reply:
yes you are correct, makes sense now, I think removing `None` is cleaner for sure.

Reply:
A link to https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy might be useful as a comment on this function. 

Alternatively, and probably preferably the func could take a concrete type from `"k8s.io/apiserver/pkg/apis/audit"`

Reply:
Lastly would we want to have a warning log in the case where an invalid option has been provided perhaps?

Reply:
I agree with a warning log albeit k8s way would be to update the status of "some" resource (surely an overkill 😝) 
I did think of a warning log but the rest of the code seems to not warn anything 🤔 ... I will add a log message.

Reply:
@PhilipGough shall I mark this as resolved?